### PR TITLE
Voucherify npaw docs

### PR DESCRIPTION
**Pull request by BC7**
# Pull Request/Issue Resolution

**Description of Change:**
Adds Voucherify and NPAW technical partner documentation.

**Reason for Change:**
Documents non-existent.

Closes #**ISSUE_NUMBER_HERE**
N/A

---
---

## PR Checklist
- [x] Ensure you have completed our CLA.
- [x] Tag @EmilyNecciai as a reviewer when the your work is done and ready to be reviewed for merge. 
- [x] Consult the [Docs Text Formatting Guide](https://github.com/Appboy/success/wiki/Docs-Text-Formatting-Guide) to be sure you're utilizing the proper markdown formatting.
- [x] Consult the [Docs Writing Style Guide & Best Practices](https://github.com/Appboy/success/wiki/Writing-Style-Guide-&-Best-Practices) to be sure you're aligning with our voice and other style best practices.
- [ ] [Preview your deployed changes](https://homeslice.braze.com/docs) to confirm that none of your changes break production. Pay close attention to links and images.
- [x] Tag others as Reviewers as necessary.
- [x] If you have modified any links, be sure to add redirects to `config/nginx.conf.erb`.

---
---

### CODE REVIEW

**Comment by BC7** [comment]
Hi @EmilyNecciai ! I'm trying to preview the deployed changes but I'm getting an error when I navigate to heroku saying:  
>An unhandled lowlevel error occurred. The application logs may have details. 

But whenever you have time to review, I think this is ready. :)

**Comment by EmilyNecciai** [comment]
@BC7 Hm, I think it means you're missing an image.

**Comment by BC7** [comment]
@EmilyNecciai I thought so too but their all linked and included correctly. I'll try to move them from img_archive to img and see if that helps.

File: _docs/_partners/technology_partners/channel_extensions/loyalty/voucherify.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+
+
+# Prerequisites
+Requirement   |Origin| Description
+--------------|------|-------------
+Braze API Key    |[Braze Settings](https://dashboard.braze.com/sign_in)| A REST API Key linked to your Braze account with the **users.track** permission enabled (**only** needed to publish via custom attributes)
+
+# Integration
+## Publish Vouchers via Connected Content
+To publish a voucher through Braze's connected content, add the following code snippet to your email template:
```
Comment (zzhaobraze) [inline _docs/_partners/technology_partners/channel_extensions/loyalty/voucherify.md]: This needs to be wrapped in a raw tag liquid isn't confused.

Comment (BC7) [reply _docs/_partners/technology_partners/channel_extensions/loyalty/voucherify.md]: That's what's going on! Thank you @zzhaobraze !

**Comment by BC7** [comment]
@EmilyNecciai Zeyuan helped me figure out what was going on! :)

**Comment by EmilyNecciai** [comment]
@cla-bot check

**Comment by EmilyNecciai** [comment]
@BC7 Looks great!! Thank you so much!

**Pull request merged**